docs/studios/unity/tutorials/first_tutorial.md

```diff
@@ -344,9 +383,60 @@ export async function processTick(rg) {
 ```
 
 The bot will find all entities of the `PowerUp` type, and then queue the action `MoveToPosition` to
-move towards that location. Push your changes, start up the scene, and then select and start the bot
+move towards that location. Push your changes, start up the scene in Unity, and then select and start the bot
 from the Regression Games overlay (click the button in the bottom right of the screen).
 
-After a few seconds, you will see your bot spawn and collect the power up!
+After a few seconds, you will see your bot spawn and collect the power up! If something is not working,
+send us a message on Slack or Discord and we will respond almost immediately!
+
+![Video of the bot performing.](first_bot_tutorial_images/tutorial_8_demo.gif)
+
+## Extensions - Configuring your bot
+
+You'll notice that in our JavaScript bot code, there is a `configureBot(rg)` function which allows
+us to set arbitrary data that is passed to Unity to configure the bot. Let's extend our Unity integration
+to take this `speed` configuration and set that on the bot in our code.
+
+First, create a new file called `BotCharacterConfig.cs` within `RGScripts/` with the following contents.
+This serialize type should match the data being returned by your bot's `configureBot` function, which
+is serialized into JSON and later parsed using this type.
+
+```cs
+using System;
+
+[Serializable]
+public class BotCharacterConfig
+{
+    public float speed;
+}
+```
+
+Now, within `CharacterBotSpawnManager.cs`, under the comment that says `// We will add more code here later to configure the bot further`
+and before `return bot`, insert the following code:
+
+```cs
+RGPlayerMoveAction moveAction = bot.GetComponent<RGPlayerMoveAction>();
+BotCharacterConfig config = botInformation.ParseCharacterConfig<BotCharacterConfig>();
+if (config != null)
+{
+    Debug.Log($"Changed speed to ${config.speed}");
+    moveAction.speed = config.speed;
+}
+return bot;
```

Comment:
```suggestion
```
Shouldn't `return bot` already be in the function already? (So we can remove this line.) The directions also reference that the code should go before 'return bot`